signerv2/README.md

```diff
@@ -1,3 +1,60 @@
-# Signerv2
+# Signer v2
 
-TODO
\ No newline at end of file
+Signerv2 is a module for signing messages. It provides a simple and unified way to produce cryptographic signatures.
+Signers instantiated from this module is required to create some SDK transaction managers (see [`NewPrivateKeyWallet`](../chainio/clients/wallet/privatekey_wallet.go) and [`NewSimpleTxManager`](../chainio/txmgr/simple.go)/[`NewGeometricTxnManager`](../chainio/txmgr/geometric/geometric.go)).
+
+## Features
```

Comment:
I think it would be beneficial to include examples for each of the different signing mechanisms, would provide better context for devs wanting to use our signer!

Reply:
Sure! We can tackle that in another PR. I created #440 to track it.